ui/selectors/first-time-flow.js

```diff
@@ -53,7 +53,7 @@ export function getFirstTimeFlowTypeRouteAfterUnlock(state) {
       : ONBOARDING_METAMETRICS;
   } else if (firstTimeFlowType === FirstTimeFlowType.socialImport) {
     if (getBrowserName() === PLATFORM_FIREFOX) {
-      return ONBOARDING_PIN_EXTENSION_ROUTE;
+      return ONBOARDING_DOWNLOAD_APP_ROUTE;
```

Comment:
do we want to show this for srp users as well?
@nakulmk

Reply:
yes, we show this also for SRP flows. the other cases/flows on this if else chains either redirect to metametrics or completion page then end up to DOWNLOAD_APP page

Metametrics > Completion > Download app > Pin Extension